packages/ember-data/tests/integration/relationships/belongs_to_test.js

```diff
@@ -451,7 +463,7 @@ test("relationship changes shouldn’t cause async fetches", function() {
   env.adapter.deleteRecord = function(store, type, record) {
     ok(record instanceof type);
     equal(record.id, 1, 'should first comment');
-    return record;
+    return record.toJSON({ includeId: true });
```

Comment:
An adapter is suppose to return JSON not a record instance. Before the meta change this worked ok because if you tried to read all the keys out of a record instance you would get the id and undefined for every other key. Not that computed properties are stored directly on the record object instead of the meta object [this code](https://github.com/emberjs/data/blob/master/packages/ember-data/lib/system/store.js#L1788) gets the computed property descriptor which blows up later when it tries to normalize it into a relationship.